docs_app/content/guide/observer.md

```diff
@@ -29,13 +29,13 @@ const observer = {
 };
 ```
 
-When subscribing to an Observable, you may also just provide the callbacks as arguments, without being attached to an Observer object, for instance like this:
+When subscribing to an Observable, you can also provide the callbacks as anonymous function, without wrapping it in an object:
```

Comment:
Whether or not you pass an anonymous function to the argument, does not matter, i don't think.  https://rxjs-ftnxyh.stackblitz.io

Could we consider rewording to "pass a references to callback functions as the arguments"?

Reply:
good point. Thanks a lot for your input on that. Happy to use this wording